homeassistant/components/zwave.py

```diff
@@ -53,7 +63,7 @@
 COMMAND_CLASS_METER = 50
 COMMAND_CLASS_BATTERY = 128
 COMMAND_CLASS_ALARM = 113  # 0x71
-COMMAND_CLASS_THERMOSTAT_MODE = 64  # 0x40
+COMMAND_CLASS_THERMOSTAT_SETPOINT = 67  # 0x43
```

Comment:
Not sure what happened there with the command classes, I didn't actually touch those myself. Git:1, Me: 0.